Source/Scene/PointPrimitiveCollection.js

```diff
@@ -0,0 +1,976 @@
+/*global define*/
+define([
+        '../Core/AttributeCompression',
+        '../Core/BoundingSphere',
+        '../Core/Cartesian2',
+        '../Core/Cartesian3',
+        '../Core/Color',
+        '../Core/ComponentDatatype',
+        '../Core/defaultValue',
+        '../Core/defined',
+        '../Core/defineProperties',
+        '../Core/destroyObject',
+        '../Core/DeveloperError',
+        '../Core/EncodedCartesian3',
+        '../Core/IndexDatatype',
+        '../Core/Math',
+        '../Core/Matrix4',
+        '../Core/PrimitiveType',
+        '../Renderer/BufferUsage',
+        '../Renderer/DrawCommand',
+        '../Renderer/ShaderSource',
+        '../Renderer/VertexArrayFacade',
+        '../Shaders/PointPrimitiveCollectionFS',
+        '../Shaders/PointPrimitiveCollectionVS',
+        './PointPrimitive',
+        './BlendingState',
+        './Pass',
+        './SceneMode'
+    ], function(
+        AttributeCompression,
+        BoundingSphere,
+        Cartesian2,
+        Cartesian3,
+        Color,
+        ComponentDatatype,
+        defaultValue,
+        defined,
+        defineProperties,
+        destroyObject,
+        DeveloperError,
+        EncodedCartesian3,
+        IndexDatatype,
+        CesiumMath,
+        Matrix4,
+        PrimitiveType,
+        BufferUsage,
+        DrawCommand,
+        ShaderSource,
+        VertexArrayFacade,
+        PointPrimitiveCollectionFS,
+        PointPrimitiveCollectionVS,
+        PointPrimitive,
+        BlendingState,
+        Pass,
+        SceneMode) {
+    "use strict";
+
+    var SHOW_INDEX = PointPrimitive.SHOW_INDEX;
+    var POSITION_INDEX = PointPrimitive.POSITION_INDEX;
+    var COLOR_INDEX = PointPrimitive.COLOR_INDEX;
+    var OUTLINE_COLOR_INDEX = PointPrimitive.OUTLINE_COLOR_INDEX;
+    var OUTLINE_WIDTH_INDEX = PointPrimitive.OUTLINE_WIDTH_INDEX;
+    var PIXEL_SIZE_INDEX = PointPrimitive.PIXEL_SIZE_INDEX;
+    var SCALE_BY_DISTANCE_INDEX = PointPrimitive.SCALE_BY_DISTANCE_INDEX;
+    var TRANSLUCENCY_BY_DISTANCE_INDEX = PointPrimitive.TRANSLUCENCY_BY_DISTANCE_INDEX;
+    var NUMBER_OF_PROPERTIES = PointPrimitive.NUMBER_OF_PROPERTIES;
+
+    var attributeLocations = {
+        positionHighAndSize : 0,
+        positionLowAndOutline : 1,
+        compressedAttribute0 : 2,        // color, outlineColor, pick color
+        compressedAttribute1 : 3,        // show, translucency by distance, some free space
+        scaleByDistance : 4
+    };
+
+    /**
+     * A renderable collection of glPoints.
+     * <br /><br />
+     * PointPrimitives are added and removed from the collection using {@link PointPrimitiveCollection#add}
+     * and {@link PointPrimitiveCollection#remove}.  PointPrimitives in a collection automatically share textures
```

Comment:
Remove reference to textures.  In general, you might want to scrub the doc to see if there are other leftovers from billboards.